components/integrations_marketplace/marketplace_modal.jsx

```diff
@@ -141,12 +143,24 @@ export default class MarketplaceModal extends React.Component {
             </div>
         );
 
+        let errorBanner = null;
+        if (this.props.serverError) {
+            errorBanner = (
+                <div
+                    className='error-bar'
+                    dangerouslySetInnerHTML={{__html: Markdown.format(localizeMessage(this.props.serverError.server_error_id,
+                        'Error connecting to the marketplace server.  Please check your settings in the [System Console](/admin_console/plugins/plugin_management).'))}} 
```

Comment:
has double spacing between `server.` and `Please check`

Reply:
Fixed